integration-tests/cucumber-tests-platform-publiclighting/src/test/java/org/opensmartgridplatform/cucumber/platform/publiclighting/mocks/oslpdevice/MockOslpChannelHandler.java

```diff
@@ -262,6 +220,51 @@ public void channelRead0(final ChannelHandlerContext ctx, final OslpEnvelope mes
         }
     }
 
+
+    private void sendNotifications(final InetSocketAddress inetAddress, final OslpEnvelope message,
+            final String deviceUid, final DeviceState deviceState) throws DeviceSimulatorException, IOException {
+
+        final MessageType messageType = this.getMessageType(message.getPayloadMessage());
+
+        if (MessageType.SET_LIGHT == messageType) {
+            final List<Oslp.LightValue> lightValueList = message.getPayloadMessage()
+                    .getSetLightRequest()
+                    .getValuesList();
+            for (final Oslp.LightValue lightValue : lightValueList) {
+                final Oslp.Event event = lightValue.getOn() ? Oslp.Event.LIGHT_EVENTS_LIGHT_ON
+                        : Oslp.Event.LIGHT_EVENTS_LIGHT_OFF;
+                final OslpEnvelope notification = buildNotification(message, deviceUid, deviceState, event,
+                        lightValue.getIndex());
+                this.send(inetAddress, notification, deviceUid);
+            }
+        }
+    }
+
+
+    private OslpEnvelope buildNotification(final OslpEnvelope message, final String deviceUid,
+            final DeviceState deviceState, final Oslp.Event event, final ByteString index) {
+        this.devicesContext.getDeviceState(deviceUid).incrementSequenceNumber();
+
+        final Oslp.EventNotification.Builder eventNotificationBuilder = Oslp.EventNotification.newBuilder()
+                .setEvent(event)
+                .setDescription(event.name())
+                //.setTimestamp(PlatformDefaults.TIMESTAMP)
```

Comment:
Default timestamp ="now", but cannot be parsed by OSGP. What is the correct timestamp format? (new Date() doesn't work) The former cucumber scenario also didn't add the timestamp to the notification.